lib/ansible/modules/command.py

```diff
@@ -21,7 +21,7 @@
        like C("*"), C("<"), C(">"), C("|"), C(";") and C("&") will not work.
        Use the M(ansible.builtin.shell) module if you need these features.
      - To create C(command) tasks that are easier to read than the ones using space-delimited
-       arguments, pass parameters using the C(args) L(task keyword,../reference_appendices/playbooks_keywords.html#task)
+       arguments, pass parameters using the C(args) L(task keyword,../../../reference_appendices/playbooks_keywords.html#task)
```

Comment:
this should not be a relative link at all, as rendering in any context outside of docs.ansible.com will create a broken link (i.e galaxy.ansible.com)

Reply:
Thanks ! no luck for me, I tried to correct the only link which wasn't correct 😄